admin_guide/install/docker_registry.adoc

```diff
@@ -76,29 +76,13 @@ For production use, you should link:../persistent_storage_nfs.html[use
 persistent storage] using `*PersistentVolume*` and `*PersistentVolumeClaim*`
 objects for storage for the registry.
 
-For non-production use, you can use the `--mount-host` option to specify a
-directory for the registry to use for persistent storage:
+For non-production use, you can use the `--mount-host=<path>` option to specify
+a directory for the registry to use for persistent storage. The registry volume
+is then created as a host-mount at the specified `<path>`.
 
-ifdef::openshift-origin[]
-----
-$ oadm registry --config=admin.kubeconfig \
-    --credentials=openshift-registry.kubeconfig \
-    --mount-host=<path>
-----
-endif::[]
-ifdef::openshift-enterprise[]
-----
-$ oadm registry --config=/etc/openshift/master/admin.kubeconfig \
-    --credentials=/etc/openshift/master/openshift-registry.kubeconfig \
-    --images='registry.access.redhat.com/openshift3/ose-${component}:${version}' \
-    --mount-host=<path>
-----
-endif::[]
```

Comment:
Removed the above because it's repeated later w/ `--service-account` which is what we're actually recommending.